environment.yml

```diff
@@ -20,4 +20,7 @@ dependencies:
   - xgboost
   - pip:
     - datasetsforecast
+    - fugue[ray]
```

Comment:
There's already a fugue in the conda section, can you remove that one?

Reply:
Yes, sure. `fugue[ray]` install other libraries for Ray and fugue to run properly. I tried integrating them in the conda section but some wheels are unavailable for py3.7. Should I include them separately in the pip section?

Reply:
I meant remove the ray from the conda section and leave this one

Reply:
hahaha I see, sorry. Fixed.